### DISCUSSION TITLE
Clustering - does each sequence get assigned at most one cluster?

### DISCUSSION THREAD

**Discussion by sgoodrow** [bbuchfink/diamond#792] [Q&A] [answered]
Hi, I'm working on an application using clustering and before I get too far I was hoping someone could clarify how the clustering algorithm works.

In particular, after running the recluster algorithm, does each sequence get assigned to exactly one cluster, or sometimes 0/2+?

Thanks!

**Comment by bbuchfink** [comment, marked as answer]
Each sequence is always assigned to exactly one cluster, regardless of using recluster or not.
